concepticondata/conceptlists/Hsiu-2015-315.tsv

```diff
@@ -0,0 +1,316 @@
+Hsiu-2015-315-84	84	dog	2009	DOG
+Hsiu-2015-315-85	85	door	1567	DOOR
+Hsiu-2015-315-86	86	dove	1853	DOVE
+Hsiu-2015-315-87	87	dream	2374	DREAM
+Hsiu-2015-315-88	88	drink water		
+Hsiu-2015-315-89	89	dry	1398	DRY
+Hsiu-2015-315-90	90	duck	1360	DUCK
+Hsiu-2015-315-91	91	dust	2	DUST
+Hsiu-2015-315-92	92	ear	1247	EAR
+Hsiu-2015-315-93	93	eat	1336	EAT
+Hsiu-2015-315-94	94	eat rice		
+Hsiu-2015-315-95	95	egg	744	EGG
+Hsiu-2015-315-96	96	elbow	981	ELBOW
+Hsiu-2015-315-97	97	embers	67	EMBERS
+Hsiu-2015-315-98	98	evening	1629	EVENING
+Hsiu-2015-315-99	99	eye	1248	EYE
+Hsiu-2015-315-100	100	face	1560	FACE
+Hsiu-2015-315-101	101	fall	1280	FALL
+Hsiu-2015-315-102	102	family	194	FAMILY
+Hsiu-2015-315-103	103	far	1406	FAR
+Hsiu-2015-315-104	104	fat		
+Hsiu-2015-315-105	105	fear	1419	FEAR (BE AFRAID)
+Hsiu-2015-315-106	106	field	212	FIELD
+Hsiu-2015-315-107	107	fire	221	FIRE
+Hsiu-2015-315-108	108	firewood	10	FIREWOOD
+Hsiu-2015-315-109	109	fish	227	FISH
+Hsiu-2015-315-110	110	floor	1632	FLOOR
+Hsiu-2015-315-111	111	flower	239	FLOWER
+Hsiu-2015-315-112	112	fly (insect)	1504	FLY (INSECT)
+Hsiu-2015-315-113	113	fog	249	FOG
+Hsiu-2015-315-114	114	forest	420	FOREST
+Hsiu-2015-315-115	115	fox	1312	FOX
+Hsiu-2015-315-116	116	fragrant	376	FRAGRANT
+Hsiu-2015-315-117	117	frog	503	FROG
+Hsiu-2015-315-118	118	fruit	1507	FRUIT
+Hsiu-2015-315-119	119	full	1429	FULL
+Hsiu-2015-315-120	120	ghost	1175	GHOST
+Hsiu-2015-315-121	121	gold	1369	GOLD
+Hsiu-2015-315-122	122	good	1035	GOOD
+Hsiu-2015-315-123	123	grass	606	GRASS
+Hsiu-2015-315-124	124	grave	1773	GRAVE
+Hsiu-2015-315-125	125	green	1425	GREEN
+Hsiu-2015-315-126	126	grow	479	GROW
+Hsiu-2015-315-127	127	gums	139	GUMS
+Hsiu-2015-315-128	128	hair, body	189	HAIR (BODY)
+Hsiu-2015-315-129	129	hair, head	2648	HAIR (HEAD)
+Hsiu-2015-315-130	130	happy	1495	HAPPY
+Hsiu-2015-315-131	131	hat	771	HAT
+Hsiu-2015-315-132	132	hay	342	HAY
+Hsiu-2015-315-133	133	head	1256	HEAD
+Hsiu-2015-315-134	134	hear	1408	HEAR
+Hsiu-2015-315-135	135	heart	1223	HEART
+Hsiu-2015-315-136	136	heavy	1210	HEAVY
+Hsiu-2015-315-137	137	hen	1514	HEN
+Hsiu-2015-315-138	138	hide	2486	HIDE
+Hsiu-2015-315-139	139	hit	1433	HIT
+Hsiu-2015-315-140	140	horse	615	HORSE
+Hsiu-2015-315-141	141	hot	1286	HOT
+Hsiu-2015-315-142	142	house	1252	HOUSE
+Hsiu-2015-315-143	143	human, person	683	PERSON
+Hsiu-2015-315-144	144	hunt, to	1435	HUNT
+Hsiu-2015-315-145	145	husband	1200	HUSBAND
+Hsiu-2015-315-146	146	I	1209	I
+Hsiu-2015-315-147	147	insect	620	INSECT
+Hsiu-2015-315-148	148	intestines	1991	INTESTINES
+Hsiu-2015-315-149	149	iron	621	IRON
+Hsiu-2015-315-150	150	jaw	798	JAW
+Hsiu-2015-315-151	151	kill	1417	KILL
+Hsiu-2015-315-152	152	knee	1371	KNEE
+Hsiu-2015-315-153	153	knife	1352	KNIFE
+Hsiu-2015-315-154	154	know	1410	KNOW (SOMETHING)
+Hsiu-2015-315-155	155	laugh	1355	LAUGH
+Hsiu-2015-315-156	156	leaf	628	LEAF
+Hsiu-2015-315-157	157	left	244	LEFT
+Hsiu-2015-315-158	158	leg	1297	LEG
+Hsiu-2015-315-159	159	lie down	215	LIE DOWN
+Hsiu-2015-315-160	160	light	737	LIGHT (IGNITE)
```

Comment:
You probably checked this one already. But we have so many "LIGHT" concepts that it could be ambiguous.